app/include/zmk/activity.h

```diff
@@ -8,4 +8,5 @@
 
 enum zmk_activity_state { ZMK_ACTIVITY_ACTIVE, ZMK_ACTIVITY_IDLE, ZMK_ACTIVITY_SLEEP };
 
-enum zmk_activity_state zmk_activity_get_state();
\ No newline at end of file
+enum zmk_activity_state zmk_activity_get_state();
+int activity_set_state(enum zmk_activity_state state);
```

Comment:
```suggestion
int zmk_activity_set_state(enum zmk_activity_state state);
```
To keep in line with the rest of our shared functions, I'm thinking this should be prefixed with `zmk_`.

Reply:
Changed